tests/testthat/test-utils.R

```diff
@@ -15,3 +15,14 @@ test_that("use_r_packages() puts listed packages on search() path", {
     expect_true(is_attached(package))
   }
 })
+
+test_that("helper skip_check_but_run_test() works as expected", {
+  # This test should not be skipped during devtools::test()
+  # This test should be skipped during devtools::check()
+  # If you remove `skip_check_but_run_test()`, devtools::check() should error
+  skip_check_but_run_test()
```

Comment:
Any test that depends on `here::here()` should pass tests but would throw an error during `devtools::check()`. That means that the checks will fail on CI. To avoid that error you can skip the problematic test by adding `skip_check_but_run_test()` inside the call to `test_that()` -- as I do here.

Happy to run a little demo.

Reply:
9accae89df39d3768067b089d296f4bb436605ad
thanks!